vpr/src/route/connection_router.cpp

```diff
@@ -431,7 +431,7 @@ void ConnectionRouter::timing_driven_expand_neighbour(t_heap* current,
                        "      Pruned expansion of node %d edge %d -> %d"
                        " (to node location %d,%dx%d,%d outside of expanded"
                        " net bounding box %d,%dx%d,%d)\n",
-                       from_node, from_node_edge_idx, to_node_int,
+                       from_node, from_edge, to_node_int,
```

Comment:
I think here, and a couple of other places we need to be calling `size_t(from_edge)` and using `%zu` in the printf pattern.

Reply:
Yep, fixed.